pkg/controller/autoscaling/elasticsearch/resources/resources.go

```diff
@@ -15,8 +15,10 @@ import (
 )
 
 const (
-	// GIB - 1 GibiByte
+	// GIB - 1 GibiBytes
```

Comment:
If 1 GiB, why a final `s`?
```suggestion
	// GIB - 1 Gibibyte
```